src/networking.c

```diff
@@ -2384,8 +2384,9 @@ parseResult handleParseResults(client *c) {
 
 /* Process the completion of an IO write operation for a client.
  * This function handles various post-write tasks, including updating client state,
+ * allow_async_writes - A flag indicating whether I/O threads can handle pending writes for this client.
```

Comment:
the flag basically state if we should offload pending writes to iothreads right?
maybe we should simply state that.
I thought of this flag as "try_send_pending_writes" but TBH it is not that important.